packages/pattern-commands/src/listeners/PluginCommandAccepted.ts

```diff
@@ -34,14 +34,12 @@ export class CommandAcceptedListener extends Listener<typeof PatternCommandEvent
 			return duration;
 		});
 
-		if (isErr(result)) {
-			message.client.emit(PatternCommandEvents.CommandError, result.error, { ...payload, duration: result.value ?? -1 });
-		}
+		result.inspectErr((error) => message.client.emit(PatternCommandEvents.CommandError, error, { ...payload, duration: -1 }));
```

Comment:
This feels wrong. We should always return a duration, even if we get an error. We should probably time outside of fromAsync

Reply:
Moved.

Reply:
wait I think this still wouldn't work for errors thrown